controller/users.go

```diff
@@ -99,21 +99,22 @@ func (c *UsersController) copyExistingKeycloakUserProfileInfo(ctx context.Contex
 		return nil, err
 	}
 
-	if existingProfile.FirstName != nil && keycloakUserProfile.FirstName != nil {
+	// If the *new* FirstName has already been set, we won't be updating it with the *existing* value
+	if existingProfile.FirstName != nil && keycloakUserProfile.FirstName == nil {
 		keycloakUserProfile.FirstName = existingProfile.FirstName
 	}
-	if existingProfile.LastName != nil && keycloakUserProfile.LastName != nil {
+	if existingProfile.LastName != nil && keycloakUserProfile.LastName == nil {
 		keycloakUserProfile.LastName = existingProfile.LastName
 	}
-	if existingProfile.Email != nil && keycloakUserProfile.Email != nil {
+	if existingProfile.Email != nil && keycloakUserProfile.Email == nil {
 		keycloakUserProfile.Email = existingProfile.Email
 	}
-	if existingProfile.Attributes != nil && keycloakUserProfile.Attributes != nil {
+	if existingProfile.Attributes != nil && keycloakUserProfile.Attributes == nil {
 		// If there are existing attributes, we overwite only those
 		// handled by the Users service in platform.
 		keycloakUserProfile.Attributes = existingProfile.Attributes
```

Comment:
Good point , i've addressed the change here :
https://github.com/almighty/almighty-core/pull/1341/files#diff-da062b312519ee2a61dd669f1ec964d4R107

Could you please verify by doing a 

```
/bin/alm-cli update users --key $ALM_TOKEN  --payload '{ "data":{ "type":"idenies" ,  "attributes":{"fullName":"johndoe JoneDow","bio":"This is my bio"}}}'  -H localhost:8080 --pp
```

We currently dont have a way to test that because we use a dummy keycloak service in tests, but will add a suite in a different PR.